index.html

```diff
@@ -713,79 +713,100 @@ <h2>Patrocínio</h2>
+                       ganham uma camiseta da Digital Ocean e Dev. Além dessa, várias outras empresas
+                       premiam colaborações específicas, como pode ser visto em
+                       <a target="_blank" href="https://hacktoberfestswaglist.com"><i class="fa fa-external-link" aria-hidden="true"></i> hacktoberfestswaglist.com</a>.
+                       Como extra, quem comparecer ao evento em Novo Hamburgo - RS estará concorrendo
+                       ao sorteio de vários brindes ao final da tarde de sábado.
                     </p>
                   </div>
                 </li>
 
                 <li>
-                  <a data-toggle="collapse" href="#faq4" class="collapsed">Ac odio tempor orci dapibus. Aliquam eleifend mi in nulla? <i class="fa fa-minus-circle"></i></a>
+                  <a data-toggle="collapse" class="collapsed" href="#faq4">Quem pode participar? <i class="fa fa-minus-circle plus"></i></a>
                   <div id="faq4" class="collapse" data-parent="#faq-list">
                     <p>
-                      Dolor sit amet consectetur adipiscing elit pellentesque habitant morbi. Id interdum velit laoreet id donec ultrices. Fringilla phasellus faucibus scelerisque eleifend donec pretium. Est pellentesque elit ullamcorper dignissim. Mauris ultrices eros in cursus turpis massa tincidunt dui.
+                      Qualquer pessoa que tenha interesse em colaborar com projetos de código aberto,
+                       independente da área de atuação ou nível de conhecimento técnico pode participar.
                     </p>
                   </div>
                 </li>
 
                 <li>
-                  <a data-toggle="collapse" href="#faq5" class="collapsed">Tempus quam pellentesque nec nam aliquam sem et tortor consequat? <i class="fa fa-minus-circle"></i></a>
+                  <a data-toggle="collapse" href="#faq5" class="collapsed">É preciso saber programar? <i class="fa fa-minus-circle plus"></i></a>
                   <div id="faq5" class="collapse" data-parent="#faq-list">
                     <p>
-                      Molestie a iaculis at erat pellentesque adipiscing commodo. Dignissim suspendisse in est ante in. Nunc vel risus commodo viverra maecenas accumsan. Sit amet nisl suscipit adipiscing bibendum est. Purus gravida quis blandit turpis cursus in
+                      Não é necessário saber programar. Há diversas formas de
+                       contribuir, inclusive com design, documentação e traduções.
                     </p>
                   </div>
                 </li>
 
                 <li>
-                  <a data-toggle="collapse" href="#faq6" class="collapsed">Tortor vitae purus faucibus ornare. Varius vel pharetra vel turpis nunc eget lorem dolor? <i class="fa fa-minus-circle"></i></a>
+                  <a data-toggle="collapse" href="#faq6" class="collapsed">Se participar eu ganho camisetas? <i class="fa fa-minus-circle plus"></i></a>
```

Comment:
This questions should be just after  _O que eu ganho ao participar?_

It is a good idea to review and better organize the questions by topic.